jax/_src/scipy/sparse/linalg.py

```diff
@@ -277,15 +274,37 @@ def cg(A, b, x0=None, *, tol=1e-5, atol=0.0, maxiter=None, M=None):
       inverse of A.  Effective preconditioning dramatically improves the
       rate of convergence, which implies that fewer iterations are needed
       to reach a given error tolerance.
+  symmetric : bool, optional
```

Comment:
On second thought, I worry that this argument name will be misleading. Without reading the details, a user might see this and think that setting `symmetric=False` will allow this algorithm to work on asymmetric real matrices.

What if we call this `assume_ipart_is_zero`? I think that's a far more descriptive name of what this parameter means here.